src/renderer/views/swap/SwapView.tsx

```diff
@@ -36,13 +84,23 @@ const SwapView: React.FC<Props> = (_): JSX.Element => {
                 .filter((a) => a.asset !== undefined && !!a.asset)
                 .map((a) => ({ asset: assetFromString(a.asset as string) as Asset, priceRune: bnOrZero(a.priceRune) }))
 
+              const hasRuneAsset = Boolean(availableAssets.find((asset) => asset.asset.symbol === PoolAsset.RUNE67C))
```

Comment:
[hint] We should use PoolAsset.RUNE67C carefully, since it's for testnet only. Issue has been filed: https://github.com/thorchain/asgardex-electron/issues/390